weldx/tags/core/file.py

```diff
@@ -187,3 +189,6 @@ def from_yaml_tree(self, node: dict, tag: str, ctx):
                     "The stored hash does not match the stored contents' hash."
                 )
         return ExternalFile(**node)
+
+    def select_tag(self, obj, tags, ctx):
+        return get_highest_tag_version(tags)
```

Comment:
I had hoped I fixed that sorting oder somewhere already... I guess not, thanks !

cref: https://github.com/asdf-format/asdf/issues/1086

can you check if
```python
    def select_tag(self, obj, tags, ctx):
        return sorted(tags)[-1]
```
also works here? `get_highest_tag_version`is certainly the safest option but also has quite the overhead

Reply:
we could add a default implementation for select_tag, right? This could be a bit fragile, if some converters are relying on a certain behaviour being changed by enforcing this.
I like the response in the linked issue, saying that asdf users are encouraged to release a new version of the extension, if some schema versions changed.

Reply:
I have implemented this in https://github.com/BAMWelDX/weldx/pull/733, no idea why I missed that when I looked into it

One only has to be careful when there are other non `weldx` tags in the list (e.g. quantity)